app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+
+  def create
+    photo = Photo.find(params[:photo_id])
+    if current_user.has_liked?(photo)
+      like = photo.likes.find(user_id: current_user.id)
+      like.destroy
```

Comment:
It's a bit weird to be destroying likes in the create method, I think it would make more sense to have the javascript send to a destroy method if there is already a like that will then destroy it